recipes/oatpp-libressl/all/test_package/oatpp-libressl-test.cpp

```diff
@@ -0,0 +1,8 @@
+#include <iostream>
+#include "oatpp-libressl/Config.hpp"
+
+int main() {
+    {        
+        oatpp::libressl::Config config;
+    }
```

Comment:
```suggestion
   oatpp::libressl::Config config;
   return 0;
```

Reply:
I'm trying to destruct object before main will ends.

Reply:
a `test_packge` is make to check the binary/headers are packaged correctly. They are not to test the life of object for the projects they wrap. This is a great minimal example and serves the purpose well but the "check" is extra for here